samples/swagger.yml

```diff
@@ -94,6 +94,28 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /bookscached/{id}:
+    get:
+      operationId: getBookByIDCached
+      description: Retrieve a book
+      x-caching:
+        max-age: 3
```

Comment:
does it make sense to make this 1 so the tests run more quickly, or does that make the tests unstable?